setup.py

```diff
@@ -12,6 +12,7 @@
         "graphviz",
         "anytree",
         "transitions",
+        "psutil",
```

Comment:
What is this used for, eventually?

Reply:
Well, this is really needed by `readout-affinity.py`, so it needs to be in the pyvenv. So it's not needed explicitly by nanorc, but we should make sure it's here.

Reply:
Actually thinking about this, this line should really go in readoutlibs